src/chatdbg/util/trim.py

```diff
@@ -65,10 +66,10 @@ def chunkify(messages, model):
 
 
 def trim_messages(
-    messages,
-    model,
+    messages: List[Dict[str, str]],
```

Comment:
This is too narrow.  Messages is a list of JSON objects that have been converted to dictionaries.